RubberduckTests/Inspections/ExcessiveInterfaceMembersInspectionTests.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using Moq;
+using Rubberduck.SettingsProvider;
+using Rubberduck.VBEditor.SafeComWrappers;
+using RubberduckTests.Mocks;
+using Rubberduck.Parsing.VBA;
+using System.Collections.Generic;
+using Rubberduck.CodeAnalysis.Inspections;
+using Rubberduck.CodeAnalysis.Inspections.Concrete;
+
+namespace RubberduckTests.Inspections 
+{
+    class ExcessiveInterfaceMembersInspectionTests : InspectionTestsBase 
+    {
+        private int AssessCode(string testCode) 
+        {
+            var modules = new List<(string, string, ComponentType)> {
+                ("Class1", testCode, ComponentType.ClassModule),
+                ("Class2", "Implements Class1", ComponentType.ClassModule) };
+
+            return InspectionResultsForModules(modules, ReferenceLibrary.Scripting).Count();
+        }
+
+        [Test]
+        [Category("Inspections")]
+        public void ExcessiveInterfaceMembers_Standard() 
+        {
+            const string testCode =
+
+                @"Public Sub Foo1()
+                End Sub
+
+                Public Sub Foo2()
+                End Sub
+
+                Public Function Foo3()
+                End Function";
```

Comment:
This VBA code is indented really weird. Note that you indentation in the code is part of the string in a verbatim string.
For consistency with other tests and readability in the text visualizer, please indent the code like VBA code from the very left of the lines.